tests/chainer_tests/test_function.py

```diff
@@ -165,101 +163,21 @@ def test_call_ndarray_gpu(self):
         self.setup_gpu()
         self.check_call_ndarray()
 
-    def check_call_volatile(self):
-        x1 = chainer.Variable(self.x1, volatile=True)
-        x2 = chainer.Variable(self.x2, volatile=True)
-        x1.rank = 1
-        x2.rank = 3
-        ys = self.f(x1, x2)
-
-        self.assertEqual(len(ys), 2)
-        self.check_check_type_forward()
-
-        for y in ys:
-            self.assertIsInstance(y, chainer.Variable)
-            self.assertEqual(y.rank, 0)
-            self.assertTrue(y.volatile)
-            self.assertIsNone(y.creator)
-
-        self.assertFalse(hasattr(self.f, 'outputs'))
-
-    def test_call_volatile_cpu(self):
-        self.check_call_volatile()
-
-    @attr.gpu
-    def test_call_volatile_gpu(self):
-        self.setup_gpu()
-        self.check_call_volatile()
-
-    def check_call_single_return_value(self, volatile):
-        x1 = chainer.Variable(self.x1, volatile=volatile)
-        x2 = chainer.Variable(self.x2, volatile=volatile)
+    def check_call_single_return_value(self):
+        x1 = chainer.Variable(self.x1)
+        x2 = chainer.Variable(self.x2)
         ret = self.f(x1, x2)
         self.assertIsInstance(ret, chainer.Variable)
 
     def test_call_single_return_value_cpu(self):
         self.f.forward_cpu.return_value = (cuda.to_cpu(self.y1),)
-        self.check_call_single_return_value(False)
+        self.check_call_single_return_value()
```

Comment:
A test for volatile flag is not required now. And, Function's behavior about backprop_mode is tested yet.